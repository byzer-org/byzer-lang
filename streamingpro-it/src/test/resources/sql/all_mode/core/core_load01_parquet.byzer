load parquet.`/ssb/customer/part-00000-7505a64d-b4bf-4186-874a-a9898502a51e-c000.snappy.parquet` as customer_parquet;

select * from customer_parquet limit 5 as customer;
