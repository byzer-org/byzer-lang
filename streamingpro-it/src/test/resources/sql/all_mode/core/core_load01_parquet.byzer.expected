C_CUSTKEY,C_NAME,C_ADDRESS,C_CITY,C_NATION,C_REGION,C_PHONE,C_MKTSEGMENT
1,Customer#000000001,j5JsirBM9P,MOROCCO  9,MOROCCO,AFRICA,25-741-298-7683,BUILDING
2,Customer#000000002,487LW1dovn6Q4dMVym,JORDAN   7,JORDAN,MIDDLE EAST,23-687-366-7194,AUTOMOBILE
3,Customer#000000003,fkRGN8n,ARGENTINA6,ARGENTINA,AMERICA,11-748-336-1281,AUTOMOBILE
4,Customer#000000004,4u58h f,EGYPT    0,EGYPT,MIDDLE EAST,14-190-594-7505,MACHINERY
5,Customer#000000005,hwBtxkoBF qSW4KrI,CANADA   7,CANADA,AMERICA,13-942-636-1142,HOUSEHOLD